debian/control

```diff
@@ -18,6 +18,7 @@ Multi-Arch: same
 Section: libs
 Depends: ${shlibs:Depends},
          ${misc:Depends},
+         wiringpi,
```

Comment:
Is this still needed?

Reply:
Yes, I execute wiringPi by now by QProcess